dask_planner/Cargo.toml

```diff
@@ -9,12 +9,12 @@ edition = "2021"
 rust-version = "1.62"
 
 [dependencies]
-arrow = { version = "28.0.0", features = ["prettyprint"] }
-async-trait = "0.1.64"
-datafusion-common = "15.0.0"
-datafusion-expr = "15.0.0"
-datafusion-optimizer = "15.0.0"
-datafusion-sql = "15.0.0"
+async-trait = "0.1.63"
```

Comment:
Do we specifically need an older version of `async-trait` or is this just a result of a bad merge with main?